src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -41,59 +41,45 @@ public function requiresExplicitTraceAnalyticsEnabling()
      */
     public function init()
     {
-        $integration = $this;
+        $rootSpan = \DDTrace\root_span();
+        if (null == $rootSpan) {
+            return Integration::NOT_LOADED;
+        }
```

Comment:
I don't understand. What does this have to do whether or not there is a root span?

Reply:
Me neither. I just ported the before state over:
https://github.com/DataDog/dd-trace-php/blob/8fee58ff83610b74e21631c5473fb96e19bcf14b/src/Integrations/Integrations/Symfony/SymfonyIntegration.php#L78-L81